### PR TITLE
Fix #91 - wrong parameter name on surrogate WriteTo.Logger configuration method

### LINKED ISSUE #91: Updated from 2.5.0 to 2.6.0 broke my project.
Asp.Net Core

Error: System.InvalidOperationException: 'No LoggingLevelSwitch has been declared with name "Debug". You might be missing a section "LevelSwitches":{"Debug":"InitialLevel"}' 

```csharp
.UseSerilog((hostingContext, loggerConfiguration) => {
    loggerConfiguration.ReadFrom.Configuration(hostingContext.Configuration);
});
```

```json
  "Serilog": {
    "MinimumLevel": {
      "Default": "Debug",
      "Override": {
        "Microsoft": "Warning",
        "System": "Warning",
        "Microsoft.AspNetCore.Authentication": "Information"
      }
    },
    "WriteTo:ConsoleSublogger": {
      "Name": "Logger",
      "Args": {
        "configureLogger": {
          "WriteTo": [
            {
              "Name": "LiterateConsole",
              "Args": {
                "outputTemplate": "DevConf [{Timestamp:HH:mm:ss} {Level}] {SourceContext}{NewLine}{Message:lj}{NewLine}{Exception}{NewLine}"
              }
            }
          ]
        },
        "restrictedToMinimumLevel": "Debug"
      }
    },
    "Enrich": [ "FromLogContext", "WithMachineName", "WithThreadId" ],
    "Properties": {
      "Application": "WashuIdentityServer"
    }
  },
```

I dont see anywhere that I tell it to use a switch. Also, I dont see this on the README and I don't see it in the samples.

### PR DESCRIPTION
**Pull request by nblumhardt**
Tactical fix for master, will need to come back onto dev.

### CODE REVIEW

File: src/Serilog.Settings.Configuration/Settings/Configuration/ConfigurationReader.cs [hunk 1]
```diff
@@ -380,8 +380,12 @@ internal static LoggerConfiguration FromLogContext(LoggerEnrichmentConfiguration
             => loggerEnrichmentConfiguration.FromLogContext();
 
         // Unlike the other configuration methods, Logger is an instance method rather than an extension.
-        internal static LoggerConfiguration Logger(LoggerSinkConfiguration loggerSinkConfiguration, Action<LoggerConfiguration> configureLogger, LoggingLevelSwitch restrictedToMinimumLevel = null)
-            => loggerSinkConfiguration.Logger(configureLogger, levelSwitch: restrictedToMinimumLevel);
```
Comment (nblumhardt) [inline src/Serilog.Settings.Configuration/Settings/Configuration/ConfigurationReader.cs]: `levelSwitch: restrictedToMinimumLevel` is the cause of the chaos.

**Comment by nblumhardt** [comment]
Tested using the sample app locally; unless anyone's around for a quick sanity check, I'll push this through once CI passes.

**Comment by nblumhardt** [comment]
Build is good; ignoring the build that only increments the package version.

File: src/Serilog.Settings.Configuration/Settings/Configuration/ConfigurationReader.cs (hunk 1 above)
Comment (tsimbalar) [reply src/Serilog.Settings.Configuration/Settings/Configuration/ConfigurationReader.cs]: oh man :-/ sorry about thus one

Comment (nblumhardt) [reply src/Serilog.Settings.Configuration/Settings/Configuration/ConfigurationReader.cs]: Easy to miss! (I did when reviewing it, anyway! :-))

**Pull request merged**